settings/settings.go

```diff
@@ -72,10 +72,12 @@ func (s Settings) PersistentDiskSettings(diskID string) (DiskSettings, bool) {
 			if hashSettings, ok := settings.(map[string]interface{}); ok {
 				diskSettings.Path = hashSettings["path"].(string)
 				diskSettings.VolumeID = hashSettings["volume_id"].(string)
+				diskSettings.ID = hashSettings["id"].(string)
```

Comment:
We use diskSettings.ID to store disk CID on line 73. It is better to not share the same property for disk ID and disk CID.